### PR TITLE
This should fix the missing option '--output-path' on webpack.

### PR DESCRIPTION
**Pull request by not-only-code**
Previous PR to `webpack` branch: https://github.com/angular/angular-cli/pull/1595
Related issue: https://github.com/angular/angular-cli/issues/1569

### CODE REVIEW

**Comment by TheLarkInn** [comment]
@not-only-code can you update your branch with master.

**Comment by not-only-code** [comment]
@TheLarkInn updated.

File: addon/ng2/tasks/serve-webpack.ts [hunk 1]
```diff
@@ -15,7 +15,8 @@ module.exports = Task.extend({
     let lastHash = null;
     let webpackCompiler: any;
 
-    var config: NgCliWebpackConfig = new NgCliWebpackConfig(this.project, commandOptions.target, commandOptions.environment).config;
+    var config: NgCliWebpackConfig = new NgCliWebpackConfig(this.project, commandOptions.target, commandOptions.environment, commandOptions.outputPath).config;
```
Comment (TheLarkInn) [inline addon/ng2/tasks/serve-webpack.ts]: Content base property for webpack dev server needs to be set also. This I believe is pulled in from webpack-build-utils.

Comment (not-only-code) [inline addon/ng2/tasks/serve-webpack.ts]: After some research, I saw that setting up `contentBase` in 'webpack-dev-server' configuration does not bundle the files into the desired folder. So i did use this plugin [write-file-webpack-plugin](https://github.com/gajus/write-file-webpack-plugin) which forces 'webpack-dev-server' to write a bundle folder.

**Comment by not-only-code** [comment]
Let me add some tests to check if all is working properly.

**Comment by TheLarkInn** [comment]
We do not want files emitted for serve. webpack-dev-server serves virtual bundles in memory. Setting content base just ensures that the dev server is point to the correct 'virtual output path' to know how to emulate the output files.

**Comment by not-only-code** [comment]
Ok, reverted last changes, just added `contentBase` variable.

**Comment by TheLarkInn** [comment]
Otherwise you might as well be using ng build with watch flag and additional server setup. In memory fs is really an important build performance feature in this case.

**Comment by TheLarkInn** [comment]
Perfect no worries and yay for tests we like that!!!!

**Comment by not-only-code** [comment]
Updated.

**Comment by TheLarkInn** [comment]
LGTM.

**Comment by TheLarkInn** [comment]
Thank you! @not-only-code

**Pull request merged**